skripchenkoveta/hometask-5/4-7.sql

```diff
@@ -0,0 +1,2 @@
+ UPDATE film SET rating = 'PG-13' length = 143 WHERE title = 'The Great Gatsby';
+ 
```

Comment:
oops what's with the line?..

Reply:
haha, i'm sorry
now I don't have a rubocop, so I fix it myself from memory:)